packages/core/src/lib/ShortcutInfo.ts

```diff
@@ -48,9 +48,9 @@ export class ShortcutInfo {
 
   toString(index: number): string {
     return `[name:'${escapeGradleString(this.name)}', ` +
-      `short_name:'${escapeGradleString(this.shortName)}', ` +
+      `shortName:'${escapeGradleString(this.shortName)}', ` +
       `url:'${this.url}', ` +
-      `icon:'${this.assetName(index)}']`;
+      `chosenIconUrl:'${this.assetName(index)}']`;
```

Comment:
Is there a reason to change `icon` -> `chosenIconUrl`? This will break the code [here](https://github.com/GoogleChromeLabs/bubblewrap/blob/81adecf07002fbd55451558257bb86242d96066a/packages/core/template_project/app/build.gradle#L179). I think this is fine to be called just `icon` at his point, as `chosenIconUrl` is more of an implementation detail then.